scripts/zkai_attention_kv_d8_bounded_softmax_table_native_gate.py

```
@@ -0,0 +1,547 @@
+        "claim_boundary": CLAIM_BOUNDARY,
+        "first_blocker": FIRST_BLOCKER,
+        "timing_policy": TIMING_POLICY,
+        "non_claims": list(NON_CLAIMS),
+        "validation_commands": list(VALIDATION_COMMANDS),
+    }
+    for key, value in expected.items():
+        if payload.get(key) != value:
+            raise AttentionKvBoundedSoftmaxTableNativeGateError(f"{key} drift")
+    receipt = payload.get("bounded_softmax_table_receipt")
+    if not isinstance(receipt, dict):
+        raise AttentionKvBoundedSoftmaxTableNativeGateError("missing bounded Softmax-table receipt")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)